apps/hash-graph/src/subcommand/server.rs

```diff
@@ -59,6 +65,38 @@ impl TryFrom<ApiAddress> for SocketAddr {
     }
 }
 
+#[derive(Debug, Clone, Parser)]
+pub struct RpcAddress {
+    /// The host the RPC client is listening at.
+    #[clap(long, default_value = "127.0.0.1", env = "HASH_GRAPH_RPC_HOST")]
+    pub rpc_host: String,
+
+    /// The port the RPC client is listening at.
+    #[clap(long, default_value_t = 25489, env = "HASH_GRAPH_RPC_PORT")]
```

Comment:
This port has been expertly chosen by a random number generator. Thought that would be better than putting it adjacent to 4001 or something similar, as that range is usually quite contended with dev servers and such.

Reply:
We can probably use `4002`

Reply:
🤝 

[5289323](https://github.com/hashintel/hash/pull/5610/commits/52893235939b74642d9064b2f5b8ac30e6c82898)